plugins/chinesehoroscope.py

```diff
@@ -0,0 +1,94 @@
+# Plugin by Infinity - <https://github.com/infinitylabs/UguuBot>
+
+import requests
+from bs4 import BeautifulSoup
+from sqlalchemy import Table, String, Column, select
+
+from cloudbot import hook
+from cloudbot.util import database
+
+table = Table(
+    'horoscope',
+    database.metadata,
+    Column('nick', String, primary_key=True),
+    Column('sign', String)
```

Comment:
Please make stored sign settings are per network instead of global

Reply:
How do I change the above to make that the case @linuxdaemon?

Reply:
Would `Column('conn', String)` work?